PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -68,9 +70,7 @@
 )
 
 
-lhcInfoTable = cms.EDProducer("LHCInfoProducer",
-    precision = cms.int32(10),
```

Comment:
this can be also removed from nanogen_cff.py

Reply:
I don't see `LHCInfoProducer ` in nanogen

Reply:
My bad, I misread `lhcInfoTable` and `lheIntoTable`